tests/assets/plugins/taggers/plugin_trainable_tagger.py

```diff
@@ -53,6 +53,9 @@ class TestTrainableTaggerModel(TrainableModel[EmptyConfig]):
       In this example model, the `from_disk(path: Path)` method provides this functionality.
     """
 
+    def train_status(self, input: TrainStatusPluginInput) -> TrainPluginOutput:
+        return TrainPluginOutput(training_complete=True)
+
```

Comment:
Thanks for this - sorry I merged the async training stuff without fixing the tests.